_posts/en/newsletters/2023-06-14-newsletter.md

```diff
@@ -0,0 +1,190 @@
+      the node _reacts_ to an incoming connection request.
+      Outbound nodes can be evicted, but that's done with an entirely
+      separate algorithm."
+  q0link="https://bitcoincore.reviews/27600#l-33"
+
+  q1="What is the impact of the `force` parameter of `SelectNodeToEvict()`
+      on the return value?"
+  a1="Specifying `force` as `true` ensures that a non-`noban` inbound peer
+      is returned, if one exists, even if it would otherwise be protected
+      from eviction.
+      Without the PR, it would not return a peer if they all are excluded
+      (protected) from eviction."
+  q1link="https://bitcoincore.reviews/27600#l-70"
+
+  q2="How is the function signature of `EraseLastKElements()` changed in this PR?"
+  a2="It changed from being a `void` return function to returning the last
+      entry that was _removed_ from the eviction candidates list. (This
+      \"protected\" node might be evicted if necessary.)
+      However, as a result of discussion during the review club meeting,
+      the PR was later simplified such that this function is no longer modified."
+  q2link="https://bitcoincore.reviews/27600#l-126"
+
+  q3="`EraseLastKElements` used to be a templated function, but this PR removes
+      the two template arguments. Why? Are there any downsides to this change?"
+  a3="This function was and (with this PR) is being called with unique template
+      arguments, so there is no need for the function to be templated.
+      The PR's changes to this function were reverted, so it's still templated,
+      because changing this would be beyond the scope of the PR."
+  q3link="https://bitcoincore.reviews/27600#l-126"
+
+  q4="Suppose we pass a vector of 40 eviction candidates to `SelectNodeToEvict()`.
+      Before and after this PR, what’s the theoretical maximum of Tor nodes
+      that can be protected from eviction?"
+  a4="With and without the PR, the number would be 34 out of 40, assuming
+      they're not `noban` and inbound."
+  q4link="https://bitcoincore.reviews/27600#l-156"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- Core Lightning 23.05.1 FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27501][] adds a `getprioritisedtransactions` RPC that
```

Comment:
It might be nice to link to the PR Review Club coverage we had on this PR:

https://bitcoinops.org/en/newsletters/2023/05/10/#bitcoin-core-pr-review-club